Src/WitsmlExplorer.Frontend/components/ContextMenus/ContextMenuUtils.tsx

```diff
@@ -87,56 +90,96 @@ export const onClickShowObjectOnServer = async (
     );
   } else if (isExpandableGroupObject(objectType)) {
     url = getObjectViewPath(
-      server.url,
+      targetServer.url,
       objectOnWellbore.wellUid,
       objectOnWellbore.wellboreUid,
       objectType,
       objectOnWellbore.uid
     );
   } else {
-    url = getObjectsViewPath(
-      server.url,
-      objectOnWellbore.wellUid,
-      objectOnWellbore.wellboreUid,
-      objectType
-    );
+    const dbQuery: UidMappingDbQuery = {
+      sourceServerId: connectedServer.id,
+      sourceWellId: objectOnWellbore.wellUid,
+      sourceWellboreId: objectOnWellbore.wellboreUid,
+      targetServerId: targetServer.id
+    };
+
+    const mappings = await UidMappingService.queryUidMapping(dbQuery);
```

Comment:
It was intentional at first, however it could be implemented for all cases. Since this is just show on server and in the case there is no corresponding object on wellbore with matching UID on target server, information that the object doesn't exist is shown.

Reply:
I mean `if (objectType === ObjectType.Log) {...` and `} else if (isExpandableGroupObject(objectType)) {...`. Currently you are not using the mapping for showOnServer for these objects: 
![image](https://github.com/user-attachments/assets/2d0246bb-9ee9-401e-a58b-db548f3605d7)

Reply:
added